test/tests_basic.jl

```diff
@@ -44,16 +44,20 @@ end
   ptr = Base.unsafe_convert(Ptr{MyStruct}, Ref(obj))
   pw = PointerWrapper(ptr)
   @test pw.value[] == 0.0
+  @test unsafe_load(ptr).value == 0.0
```

Comment:
What are these new tests for? Are they related to the new conversion?

Reply:
No, they are unrelated. But I thought it is good to test if the underlying data of the pointer is also changed, when changing the data from the PointerWrapper. I just saw that we didn't really have such a test before. And it is something I would like to rely on in Trixi.jl.

Reply:
Thanks! /r